src/backgrounds/_base.scss

```diff
@@ -0,0 +1,113 @@
+/**
+ * @file Backgrounds
+ * @module Backgrounds
+ * @overview Background and matching text styles for page modules
+ * @todo: remove _backdrops.scss since this replaces it?
+ * @todo: rename TrialWarranty to be generic, update to use Starry Background
+ */
+
+.background--gray {
```

Comment:
It's hard to do, but it is best to avoid creating super specific styling classes. Can we use `light` or a different key? `base`? `default`? `background--muted`? Would be great to hear what design calls it internally. Maybe we can glean a good class name from their usage. Unless they just call it gray :stuck_out_tongue_winking_eye: